src/PrestaShopBundle/Bridge/Helper/Listing/HelperListConfiguration.php

```diff
@@ -28,210 +28,619 @@
-    public $where = '';
+    public $listError;
 
     /**
      * @var string
      */
-    public $join = '';
+    public $orderWay = '';
+
+    /**
+     * @var int
+     */
+    private $tabId;
 
     /**
      * @var string
      */
-    public $group = '';
+    private $tableName;
 
     /**
      * @var string
      */
-    public $orderBy = '';
+    private $listId;
 
     /**
      * @var string
      */
-    public $defaultOrderBy;
+    private $objectModelClassName;
 
     /**
      * @var string
      */
-    public $orderWay = '';
+    private $legacyControllerName;
 
     /**
      * @var string
      */
-    public $defaultOrderWay = 'ASC';
+    private $identifier;
 
     /**
      * @var string
      */
-    public $having = '';
+    private $token;
 
     /**
      * @var string
      */
-    public $filterHaving = '';
+    private $defaultOrderBy;
 
     /**
      * @var string
      */
-    public $filter = '';
+    private $defaultOrderWay = 'ASC';
 
     /**
      * @var bool
      */
-    public $isJoinLanguageTableAuto = false;
+    private $autoJoinLanguageTable;
 
     /**
      * @var int
      */
-    public $defaultPagination = 50;
+    private $multiShopContext;
 
     /**
-     * @var array
+     * @var bool
      */
-    public $pagination = [20, 50, 100, 300, 1000];
+    private $deleted;
 
     /**
-     * @var string
+     * @var bool
      */
-    public $shopLinkType = '';
+    private $explicitSelect;
 
     /**
-     * @var int
+     * @var bool Use SQL_CALC_FOUND_ROWS / FOUND_ROWS to count the number of records
+     */
+    private $useFoundRows;
+
+    /**
+     * @var string
      */
-    public $multishopContext;
+    private $positionIdentifier;
 
     /**
      * @var bool
      */
-    public $deleted = false;
+    private $bootstrap;
 
     /**
-     * @var array
+     * @var string|null
      */
-    public $fieldsList = [];
+    private $legacyCurrentIndex;
+
+    /**
+     * @var array<string, mixed>
+     */
+    private $deleteLinkVars = [];
 
     /**
      * @var string
      */
-    public $listsql;
+    private $shopLinkType = '';
 
     /**
-     * @var array
+     * @var int[]
      */
-    public $list;
+    private $paginationLimits = [20, 50, 100, 300, 1000];
 
     /**
      * @var int
      */
-    public $listTotal;
+    private $defaultPaginationLimit = 50;
 
     /**
-     * @var string
+     * @var array<string, array<string, mixed>>
      */
-    public $listError;
+    private $toolbarActions = [];
+
+    /**
+     * @var string[]
+     */
+    private $rowActions = [];
+
+    /**
+     * @var array<string, array<string, mixed>>
+     */
+    private $bulkActions = [];
+
+    /**
+     * @var string|null
+     */
+    private $postSubmitUrl;
 
     /**
      * @var bool
      */
-    public $explicitSelect = false;
+    private $shopShareData = false;
 
     /**
-     * @var bool Use SQL_CALC_FOUND_ROWS / FOUND_ROWS to count the number of records
+     * @param int $tabId
```

Comment:
Please describe the parameters

Reply:
You can copy the description from the fields here